### PR TITLE
implemneted get environment

### PR DESCRIPTION
**Pull request by ilansh**
I am using this package quite a bit, would be happy to give back some.  
This change was tested with real gitlab api, made sure it didn't break ListEnvironments either.

### CODE REVIEW

File: environments.go [hunk 1]
```diff
@@ -32,10 +32,11 @@ type EnvironmentsService struct {
...
 type Environment struct {
-	ID          int    `json:"id"`
-	Name        string `json:"name"`
-	Slug        string `json:"slug"`
-	ExternalURL string `json:"external_url"`
+	ID             int        `json:"id"`
+	Name           string     `json:"name"`
+	Slug           string     `json:"slug"`
+	ExternalURL    string     `json:"external_url"`
+	LastDeployment Deployment `json:"last_deployment,omitempty"`
```
Comment (svanharmelen) [inline environments.go]: This should be a pointer (`*Deployment`) and the `omitempty` tag is not needed here.

File: environments.go [hunk 2]
```diff
@@ -74,6 +75,31 @@ func (s *EnvironmentsService) ListEnvironments(pid interface{}, opts *ListEnviro
 	return envs, resp, err
 }
 
+// GetEnvironment gets a specific environment from a project
```
Comment (svanharmelen) [inline environments.go]:
```suggestion
// GetEnvironment gets a specific environment from a project.
```

**Comment by ilansh** [comment]
Done, should I squash the commits?

**Comment by svanharmelen** [comment]
All good 👍Thanks!

**Pull request merged**